aten/src/ATen/nn.yaml

```diff
@@ -5,6 +5,12 @@
   scalar_check:
     output: 'false'
     grad_input: 'false'
+  CPU:
+    forward_scalar_types: ['Float', 'Double', 'Half']
+    backward_scalar_types: ['Float', 'Double', 'Half']
+  GPU:
```

Comment:
In the code below I see the backend being defined as 'CUDA':
`        backend = 'CUDA' if re.search('THCU', path) else 'CPU'`

Reply:
Fixed